Version 0.44.0 does not include limits library by its own, this fixes it.

diff -ruN kallisto_orig/src/Inspect.h kallisto/src/Inspect.h
--- kallisto_orig/src/Inspect.h 2022-09-05 15:33:04.350000000 +0200
+++ kallisto/src/Inspect.h      2022-09-05 15:33:31.070000000 +0200
@@ -3,6 +3,7 @@

 #include <iostream>
 #include <unordered_set>
+#include <limits>
 #include "KmerIndex.h"
 #include "GeneModel.h"

diff -ruN kallisto_orig/src/MinCollector.cpp kallisto/src/MinCollector.cpp
--- kallisto_orig/src/MinCollector.cpp  2022-09-05 15:33:04.350000000 +0200
+++ kallisto/src/MinCollector.cpp       2022-09-05 15:33:53.640000000 +0200
@@ -1,5 +1,6 @@
 #include "MinCollector.h"
 #include <algorithm>
+#include <limits>

 // utility functions

diff -ruN kallisto_orig/src/main.cpp kallisto/src/main.cpp
--- kallisto_orig/src/main.cpp  2022-09-05 15:33:04.350000000 +0200
+++ kallisto/src/main.cpp       2022-09-05 15:34:20.330000000 +0200
@@ -7,6 +7,7 @@
 #include <getopt.h>
 #include <thread>
 #include <time.h>
+#include <limits>

 #include <cstdio>
